### ISSUE TITLE
setting initialIndex higher than available cells causes error

### ISSUE THREAD

**Issue by Mahjouba91** [metafizzy/flickity#291]
Hello,

I loaded your library on my wordpress website like this : 

```php	
wp_register_script( 'flickity', '//cdnjs.cloudflare.com/ajax/libs/flickity/1.1.1/flickity.pkgd.min.js', array( 'jquery' ), null, true );
...
wp_enqueue_script( 'flickity' );
```
It works fine for my gallery, but on my WooCommerce products, the JS of the add-to-cart JS script isn't  executed, there is no error in the console log. When I remove flickity script, it works fine.

So do you know why there is such a bug like that ? I will try to contact Woocommerce support too.

By the way, on the homepage, I got this error (there is a gallery in this page) I tried to load the unminified to show you the right line number : 
```
flickity.pkgd.js:4220 Uncaught TypeError: Cannot set property 'className' of undefined 
flickity.pkgd.min.js:12 Uncaught TypeError: Cannot read property 'target' of undefined
```

**Comment by desandro** [comment]
Thanks for reporting this bug. Sorry to say I don't have any experience with WooCommerce so I can't identify the problem here. Do you have a live URL I could take a look at? See [Submitting Issues in the contributing guidelines](https://github.com/metafizzy/flickity/blob/master/contributing.md#submitting-issues).

**Comment by mikedidthis** [comment]
I managed to replicate part of this issue. 

I don't believe `woocommerce` is the issue, it seems to be with `pageDots` and setting an `initialIndex` higher than the amount of slides available: http://codepen.io/anon/pen/vLLrJP

I believe this is an implementation issue, rather than a plugin issue.

**Comment by desandro** [comment]
@mikedidthis Thanks so much for diagnosing this issue. I've released v1.1.2 with a fix

**Comment by mikedidthis** [comment]
@desandro No problem at all. Thank you for the rapid fix :+1:
